src/Assets/styleController.ts

```diff
@@ -48,6 +60,7 @@ function generateMuiOverides(theme: ITheme) {
   const blueColor = theme.primaryColor;
   const greenColor = theme.secondaryColor;
   const blackColor = '#2A2B2A';
+  const midBlue = '#41528C'
```

Comment:
Could you move this to themes object. We might need to change this color on our white label.